packages/analysis/vite.config.ts

```diff
@@ -0,0 +1,3 @@
+import { viteConfig as defineBaseViteConfig } from '@terra-ui-packages/build-utils';
```

Comment:
The exported name `viteConfig` isn't great. The name suggests that it is a configuration object when it's actually a function that returns the config object. I intend to rename this export in the future, but keeping it here to limit the scope of this PR.